IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -59,73 +58,105 @@ def extract_study_info(session, screen_id):
     return details_
 
 
-def describe_screen(screen_id):
+def clean_channel_item(channel_item):
+    if bool(channel_item[channel_item.find("(") + 1 : channel_item.rfind(")")]):
+        channel_item = re.sub(r"\([^()]*\)", "", channel_item)
+
+    return channel_item.strip()
+
+
+def describe_screen(screen_id, session, sample, imaging_method, study_name):
     """Pull additional metadata info per plate, given screen id
 
     Parameters
     ----------
     screen_id: int
         ID of the screen data set
+    sample: str
+        Indicates cell or tissue
+    imaging_method: str
+        Method used to collect images
+    study_name: str
+        IDR accession name
 
     Returns
     -------
     pandas.DataFrame() of the following metadata values:
 
         screen_id: IDR ID for the screen
+        study_name: IDR accession name
         plate_id: IDR ID for each plate
         plate_name: Names given to each plate
-        image_id: IDR ID for each image
+        well_id: IDR ID for each well
```

Comment:
Are the wells designated with an IDR ID? Isn't it just well location? 

Have you found this to be standardized, at least?

Reply:
@gwaybio Yes, each well has it's own id. It appears to be similar to the image id where its unique to each image. So yes, it's standardized from what I've seen of the metadata thus far.

I bet it could be denoted by well location, but using well_id I can access metadata per specific well through the API just as I can with images or plates.